packages/blade/package.json

```diff
@@ -28,7 +28,12 @@
   },
   "scripts": {
     "prepublish": "yarn build",
-    "build": "rollup -c && node ./scripts/generateRootImports.js",
+    "build": "run-s build:*",
+    "build:clean": "rm -rf build",
+    "build:generate-types": "tsc -p ./tsconfig-generate-types.json",
+    "build:copy-declarations": "copyfiles -u 1 \"src/**/*.d.ts\" build/types",
+    "build:source": "rollup -c && node ./scripts/generateRootImports.js",
```

Comment:
yes you can run them individually nothing would break, it just won't give you desired output. If I add it as a single script it'll go out of my monitor(kidding lol) but it'll just create more confusion on single line and make understanding things complicated. And anyways no one really touches these scripts esp on the DS maintainers side since it's just tooling things